tests/loggers/test_base.py

```diff
@@ -176,6 +176,32 @@ def training_step(self, batch, batch_idx):
     assert logger2.finalized_status == "success"
 
 
+def test_trainer_loggers_property(tmpdir):
```

Comment:
Missed this :p

```suggestion
def test_trainer_loggers_property():
```

Reply:
There are actually numerous tests in this file that have tmpdir as a parameter but don't use it at all. I left it in just to be safe. Should I fix the other tests here or make a separate issue for that?

Reply:
I'd say create a separate issue so that we do it for the full testbase.
It's an easy one to miss